backend-external/src/config/index.ts

```diff
@@ -16,10 +16,11 @@ config.defaults({
       windowMs: process.env.RATE_LIMIT_WINDOW_MS || 60000, // 1 minute
       limit: process.env.RATE_LIMIT_LIMIT || 100, // Limit each IP to 100 requests per `window` (here, per 1 minute)
     },
+    url: process.env.EXTERNAL_BACKEND_SERVER_URL || 'http://localhost:3002'
   },
   backend:{
     apiKey: process.env.BACKEND_EXTERNAL_API_KEY || 'api-key',
-    url: process.env.BACKEND_URL || 'http://localhost:3000'
+    url: process.env.BACKEND_URL || 'http://localhost:3010'
```

Comment:
Any particular reasons we need to make it 3010?

Reply:
The external api app in the backend is running on port 3010